### PR TITLE
CircleCI Udpate to python3.8 enviorment

### PR DESCRIPTION
**Pull request by GalRabin**
FYI - @yaakovi @reutshal 

## Status
Ready

## Related Issues
related: https://github.com/demisto/etc/issues/29577

## Description
Walrus and other python38 feature currently not supported in our content repo due to the fact that they fail when running lint in CircleCI, the next steps in order to support it after update the docker file is to replace it in config.yml and verify it on the nightly run.

Reason for failure: Mypy run locally and the interpreter we have locally in CircleCI is python37

### CODE REVIEW

File: docker/content-build/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM circleci/python:3.7.5-buster-node
+FROM circleci/3.8.6-buster-node-browsers
```
Comment (glicht) [inline docker/content-build/Dockerfile:1]: The version you put in seems wrong. It should be:
```suggestion
FROM circleci/python:3.8.6-buster-node
```

Comment (GalRabin) [reply docker/content-build/Dockerfile]: Right

**Comment by glicht** [comment]
@GalRabin why all these changes just to upgrade the python version?

**Comment by GalRabin** [comment]
> @GalRabin why all these changes just to upgrade the python version?

Due to the issue, I told you about on bootstrap which also reproduced in other environments (https://app.circleci.com/pipelines/github/demisto/content/35037/workflows/1960a872-087b-46b0-b9df-d423b54fb387/jobs/152604)

Link to issue: https://github.com/demisto/etc/issues/29637
We must find a solution to it.

**Pull request closed without merging**